### PR TITLE
22776 - Shortname History Table Fixes

### PR DESCRIPTION
**Pull request by ochiu**
*Issue #:*
https://github.com/bcgov/entity/issues/22776

*Description of changes:*
- Fix reload should go back to page 1
- Fix issue where infinite scroll back is no longer triggering if it previously has reached an end state even after reload

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by Jxio — APPROVED** [review]

File: auth-web/src/components/pay/eft/ShortNamePaymentHistory.vue [hunk 1]
```diff
@@ -152,6 +153,7 @@ import { DEFAULT_DATA_OPTIONS } from '../../datatable/resources'
 import { EFTTransactionState } from '@/models/eft-transaction'
 import ModalDialog from '@/components/auth/common/ModalDialog.vue'
 import PaymentService from '@/services/payment.services'
+import { Vue } from 'vue-property-decorator'
```
Comment (seeker25) [inline auth-web/src/components/pay/eft/ShortNamePaymentHistory.vue:156]: Not too sure why we're introducing this, we're actively trying to get away from using Vue-class-component and vue-property-decorartor

Comment (seeker25) [reply auth-web/src/components/pay/eft/ShortNamePaymentHistory.vue:156]: `import Vue from 'vue'`

Comment (ochiu) [reply auth-web/src/components/pay/eft/ShortNamePaymentHistory.vue:156]: Will fix this in my next PR, as it is another FE ticket.

Comment (seeker25) [reply auth-web/src/components/pay/eft/ShortNamePaymentHistory.vue:156]: all good, I was too slow on the review haha

**Pull request merged**